model/team.go

```diff
@@ -19,7 +19,7 @@ type Team struct {
 	UpdateAt       int64  `json:"update_at"`
 	DeleteAt       int64  `json:"delete_at"`
 	Name           string `json:"name"`
-	Domain         string `json:"domain"`
+	URLId          string `json:"urlid"`
```

Comment:
We are going to go with Name and DisplayName to be consistent with channels. Corey is 5/5 on this.